scrapers/Hustler.yml

```
@@ -39,18 +39,25 @@ sceneByURL:
     scraper: sceneScraper
 xPathScrapers:
   sceneScraper:
+    common:
+      $content: //div[@class="panel-content"]
     scene:
-      Title: //h3/a/text()
+      Title: //h3/a/text()|$content//h3[contains(@class, "headline")]/a/text()
       URL: //link[@rel='canonical']/@href
       Performers:
         Name: 
-          selector: //span[@class="attr-key" and contains(text(),"Cast")]/following-sibling::span[@class="attr-value"]/a[not(text()="Hustler Models") and not(text()="Barely Legal Models")]/text()
+          selector: //span[@class="attr-key" and contains(text(),"Cast")]/following-sibling::span[@class="attr-value"]/a[not(text()="Hustler Models") and not(text()="Barely Legal Models")]/text()|$content//span[contains(text(), "Cast")]//following-sibling::span/a/text()
       Date:
-        selector: //span[@class="attr-key" and contains(text(),"Released")]/following-sibling::span[@class="attr-value"]/text()
+        selector: //span[@class="attr-key" and contains(text(),"Released")]/following-sibling::span[@class="attr-value"]/text()|$content//span[contains(text(), "Released")]//following-sibling::span/text()
         postProcess:
           - parseDate: Jan 02, 2006
-      Details: //meta[@property="og:description"]/@content|//div[@class="description"]/p
-      Image: //div[@class="img-container"]/img/@src
+      Details: //p[following-sibling::a[@class="clickable"]]|//meta[@property="og:description"]/@content|//div[@class="description"]/p
```

```suggestion
      Details: //div[@class="panel-content"]/div/div/text()|//meta[@property="og:description"]/@content|//div[@class="description"]/p
```
gets part of the details for older scenes

---

hmm, when I try this with the new scenes, it also gets the first part (before clicking Read More) rather than the full description... will see if I can come up with a selector combo that gets the full description for all scenes